site/app/views/forum/ForumThreadView.php

```diff
@@ -612,4 +625,86 @@ public function createThread() {
 		return $return;
 	}
 
+
+	public function statPage($users) {
+
+		$return = <<<HTML
+		
+		<div style="margin-left:20px;margin-top:10px; height:50px;" id="forum_bar">
+
+			<a class="btn btn-primary" style="border:3px solid #E9EFEF" title="Back to threads" href="{$this->core->buildUrl(array('component' => 'forum', 'page' => 'view_thread'))}"><i class="fa fa-arrow-left"></i> Back to Threads</a>
+			<a class="btn btn-primary" style="border:3px solid #E9EFEF" title="Show Stats" onclick="resetScrollPosition();" href="{$this->core->buildUrl(array('component' => 'forum', 'page' => 'show_stats'))}">Stats</a>
```

Comment:
Can we remove this button since we are already on the page that it redirects to (we are already on the stats page)?